### DISCUSSION TITLE
Can I create reth's mdbx using Erigon's db or Geth's dump?

### DISCUSSION THREAD

**Discussion by debjit-bw** [paradigmxyz/reth#11995] [Help] [answered]
From #7406 I understand we cannot use geth's levelDb with reth. But why can't reth sync with Erigon (#7915) despite using the same database format? Additionally, I see reth stores a `static_files` folder with some conf, off files. What are they really?

**If reth cannot use Erigon's mdbx**, is there a way to create the `mdbx + static_files` database from a [key -> rlp] dump (from geth) or the `mdbx.dat` and `mdbx.lck` from Erigon db? I need to start the chain from a certain block in the middle (post-merge to be exact, because our impl which depends on reth isn't compatible with some pre-merge stuff), and am trying for ways to do it without implementing pre-merge stuff.

**Comment by Rjected** [comment, marked as answer]
There is no easy way to migrate the erigon or geth format DB to a reth node. Reth uses a static file format for storing immutable data, since it's not desirable to have that data in the database. Reth and erigon use the same database but do not organize data in the same way, and are incompatible as a result.
